Penteract/Assets/Scenes/LoseScene.scene

```diff
@@ -190,7 +190,10 @@
                         "Type": "AudioListener",
                         "Id": 11540703914281798444,
                         "Active": true,
-                        "Gain": 0.3019999861717224
+                        "Gain": 0.3019999861717224,
+                        "ModelIndex": 0,
```

Comment:
I forgot to say that you need to change this for each AudioListener:
DistanceModel: Linear
Clamped: True

Reply:
Okey! I will fix it!